.changeset/hip-schools-greet.md

```diff
@@ -5,7 +5,7 @@
 
 Modal:
 - Rename `Modal.Dialog` to `Modal`
-- Rename `Modal.Root` to `Modal.Context`
+- Rename `Modal.Root` to `Modal.TriggerContext`
```

Comment:
I dont think you should edit old changesets.

Reply:
heh bit fast on the search and replace here - good catch